components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/AttributeMapper.java

```diff
@@ -110,11 +112,14 @@ else if (attribute instanceof ComplexAttribute) {
     }
 
     /**
-     * set claim mapping for simple attribute
+     * set claim mapping for simple attribute.
```

Comment:
Check the capitalization for the comments in all the places.

Reply:
Fixed in https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/commit/d14dfd5420cd70c2bb864885bcb9ab336dca6ac0